### PR TITLE
Spec description grammer [docs]

### PR DESCRIPTION
**Pull request by mjago**
Small changes to improve spec doc description

### CODE REVIEW

File: src/spec.cr [hunk 1]
```diff
@@ -24,25 +24,25 @@ require "./spec/dsl"
...
+# Typically a top level `describe` defines the `outer` unit (such as a class)
+# that is to be tested by the spec. Further `describe` calls can be nested within
+# the outer unit to specify smaller units under test (such as individual methods).
+# `describe` can also be used to set up a certain context - think empty `Array` versus
+# `Array` with elements. The `context` method behaves just like the `describe` method
+# and may be used instead, to emphasize context to the reader.
 #
-# Concrete test cases are defined with `it` within a `describe` block. A
-# descriptive string is supplied to `it` describing what that test case
+# Within a `describe` block concrete test cases are defined with `it` . A
```
Comment (RX14) [inline src/spec.cr]: Comma after block?

File: src/spec.cr [hunk 2]
```diff
@@ -24,25 +24,25 @@ require "./spec/dsl"
...
+# descriptive string is supplied to `it` describing what the test case
 # tests specifically.
 #
 # Specs then use the `should` method to verify that the expected value is
-# returned, see the example above for details.
```
Comment (RX14) [inline src/spec.cr]: I'm not 100% on this change, maybe move the full stop before the brackets?

**Comment by mjago** [comment]
Thanks for suggestions :+1:

**Comment by mverzilli** [comment]
This is great! Thank you @mjago ❤️

**Pull request merged**